src/qforte/sq_operator.h

```diff
@@ -19,12 +19,8 @@ class SQOperator {
     /// default constructor: creates an empty second quantized operator
     SQOperator() {}
 
-    /// TODO: implement
-    /// build from a string of open fermion qubit operators
-    // void build_from_openferm_str(std::string op) {}
-
-    /// add one product of anihilators and/or creators to the second quantized operator
-    void add_term(std::complex<double> coeff, const std::vector<size_t>& ac_ops);
+    /// add one product of annihilators and/or creators to the second quantized operator
```

Comment:
Can we please add some text explaining on how the operators are ordered? For example, in Forte we have something like:
```
 * A data structure used to represent a second quantized operator string like
 *
 * ... op(2) op(1) op(0), where op(i) = a^\dagger_(orb_i, spin_i) or a_(orb_i, spin_i)
 * ```

Reply:
To clarify, this should probably appear at the beginning of the class.